pkg/destroy/gcp/route.go

```diff
@@ -56,6 +57,9 @@ func (o *ClusterUninstaller) deleteRoute(item cloudResource) error {
 	op, err := o.computeSvc.Routes.Delete(o.ProjectID, item.name).RequestId(o.requestID(item.typeName, item.name)).Context(ctx).Do()
 	if err != nil && !isNoOp(err) {
 		o.resetRequestID(item.typeName, item.name)
+		if strings.HasPrefix(item.name, "default-route") {
+			return errors.New("this looks like a default route, which cannot be deleted manually but will be deleted with the corresponding network")
```

Comment:
what about error here https://github.com/openshift/installer/blob/5ea1f2c28a4ea33960b1000dcff8d4959b187289/pkg/destroy/gcp/route.go#L65-L68 ??

Reply:
Hitting this error would be unexpected, so I don't see a need for special handling as in the first case where we expect to hit the error. Let me know if you disagree.